pebbles/tasks/celery_app.py

```diff
@@ -107,7 +107,12 @@ def get_dynamic_config():
         'schedule': crontab(minute='*/1'),
         'options': {'expires': 60, 'queue': 'system_tasks'},
     },
-    'periodic-update-every-month': {
+    'periodic-update-every-day': {
+        'task': 'pebbles.tasks.instance_token_cleanup',
+        'schedule': crontab(minute='*/1'),
```

Comment:
@apurva3000 Is this supposed to run every minute of the day ?

Reply:
I guess there is some error over here, will check